__tests__/posts.ts

```diff
@@ -2011,6 +2011,126 @@ describe('mutation sharePost', () => {
   });
 });
 
+describe('mutation editSharePost', () => {
+  const MUTATION = `
+  mutation editSharePost($id: ID!, $sourceId: ID!, $commentary: String) {
+    editSharePost(sourceId: $sourceId, id: $id, commentary: $commentary) {
+      id
+    }
+  }`;
+
+  const variables = {
+    sourceId: 's1',
+    id: 'sharePost',
+    commentary: 'My comment',
+  };
+
+  beforeEach(async () => {
+    await con.getRepository(SquadSource).save({
+      id: 's1',
+      handle: 's1',
+      name: 'Squad',
+      private: false,
+      memberPostingRank: 0,
+    });
+    await con.getRepository(SourceMember).save({
+      sourceId: 's1',
+      userId: '1',
+      referralToken: 'rt',
+      role: SourceMemberRoles.Member,
+    });
+    await con.getRepository(SharePost).save({
+      id: 'sharePost',
+      shortId: 'sharePost',
+      sourceId: 's1',
+      type: PostType.Share,
+      title: 'Foo Bar',
+      authorId: '1',
+    });
+  });
```

Comment:
I think this already happens on top level, we should aim to avoid re-running extra queries to avoid timings

Reply:
It was not happening at the top level actually (this file could use some cleanup overall IMO) -- but I did replace it with the existing function for generating fixtures in this file 🙏